### ISSUE TITLE
Export Homee's own status

### ISSUE THREAD

**Issue by programmieramt** [xenji/homee_exporter#9]
As user I want to know in what state(Abwesend/Urlaub/Zuhause...) is

**Comment by xenji** [comment]
This is what I have in the nodes. Need to find out what means what in regards to the state.

```json
{
    "id": -1,
    "name": "homee",
    "profile": 1,
    "image": "default",
    "favorite": 0,
    "order": 0,
    "protocol": 0,
    "routing": 0,
    "state": 1,
    "state_changed": 1492869761,
    "added": 1492869761,
    "history": 1,
    "cube_type": 0,
    "note": "",
    "services": 255,
    "phonetic_name": "",
    "owner": 1,
    "denied_user_ids": [],
    "attributes": [
        {
            "id": 1,
            "node_id": -1,
            "instance": 0,
            "minimum": 0,
            "maximum": 200,
            "current_value": 0.0,
            "target_value": 0.0,
            "last_value": 0.0,
            "unit": "n%2Fa",
            "step_value": 1.0,
            "editable": 1,
            "type": 205,
            "state": 1,
            "last_changed": 1492875739,
            "changed_by": 1,
            "changed_by_id": 0,
            "based_on": 1,
            "data": "",
            "options": []
        }
    ]
}
```

**Comment by xenji** [comment]
Type `205` is the HomeeMode, but we still need to find out the mapping via try & observe.

**Comment by xenji** [comment]
Fixed with 0.2.0
